config/locales/en.yml

```diff
@@ -265,6 +265,8 @@ en:
       link_texts:
         not_started: Start section
         in_progress: Continue section
+    school_result_notice:
+      result_text: "%{remaining_search_count} more schools match your search for ‘%{search_query}’. Try narrowing down your search if the school you're looking for is not listed."
```

Comment:
shouldn't it be plural? as x number matches

Reply:
Yeah this text will need to have a singular and plural version. Or else truncate 1 higher when there are 16 results.

Reply:
Fixed in https://github.com/DFE-Digital/register-trainee-teachers/pull/955/commits/bf2ac29f8d24ee41048f96e351f39f036d4f4c5d